src/main/kotlin/no/nav/hjelpemidler/delbestilling/delbestilling/DelbestillingModel.kt

```diff
@@ -76,6 +76,7 @@ data class DelbestillingResultat(
     val id: UUID,
     val feil: DelbestillingFeil? = null,
     val saksnummer: Long? = null,
+    val delbestillingSak: DelbestillingSak?,
```

Comment:
```suggestion
    val delbestillingSak: DelbestillingSak? = null,
```
Kan defaulte til null slik at vi ikkje trenger å legge på `..., null, null)` alle plassar i DelbestillingService